controllers/user.go

```diff
@@ -0,0 +1,56 @@
+package controllers
+
+import (
+	"fmt"
+	"net/http"
+
+	forms "go-crawler-challenge/forms/user"
```

Comment:
We dont need to add the name as it is the same as the package name
```suggestion
	"go-crawler-challenge/forms/user"
```

Reply:
@junan I declare like this because of using `forms` in here https://github.com/Lahphim/go-crawler-challenge/pull/36/files#diff-64a63228546ebdbd9a1ee7a43793f89b24047f3bd080a952d86abdd38ed2b258R41 

For my idea, I think it look more readable in code. 🤔
Or what if I change it to `form "go-crawler-challenge/forms/user"`?
So it won't redundant with the package name anymore 😅

Reply:
> Or what if I change it to form "go-crawler-challenge/forms/user"?

Ya, renaming the package is fine for me in that case

One issue just noticed, the package path should be just `go-crawler-challenge/forms`(not `go-crawler-challenge/forms/user`)

Reply:
updated https://github.com/Lahphim/go-crawler-challenge/pull/36/commits/6b4e9e74883bfbabaadfa773e7bc65798de3facf